CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.17)
+cmake_minimum_required(VERSION 3.16)
```

Comment:
Unless none of you can actually compile in 3.17 we should stick with it.

Reply:
Due to the fact that Léo and me got this problem i think it will be fine to downgrade the CMake version